sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/xds/XdsServiceDiscovery.java

```diff
@@ -29,15 +31,24 @@
  **/
 public interface XdsServiceDiscovery {
     /**
-     * get service instance by service name
+     * get all the service instance without tag by service name
      *
      * @param serviceName service name
      * @return service instances
      */
     Set<ServiceInstance> getServiceInstance(String serviceName);
```

Comment:
You are advised to change the method name to getServiceInstanceWithoutTags.

Reply:
The comment has been modified.